src/book/02-system/04-hardware/04-dynamixel-protocol.mdx

```diff
@@ -0,0 +1,388 @@
+---
+section: System
+chapter: Hardware
+title: Dynamixel Protocol
+description: Learn about the systems of the Dynamixel protocol
+slug: /system/hardware/dynamixel-protocol
+authors:
+  - Clayton Carlon (@Claegtun)
+---
+
+## Introduction
+
+Dynamixel is the name that Robotis gives to its [system](https://www.robotis.us/dynamixel/) off-the-shelf servos that have integrated control. These are such that the servos are controlled by commands over a digital interface. There are broadly three protocols for such an interface, namely ModbusRTU, Dynamixel v1, and Dynamixel v2. We will only look into the last of the three which will be simply called the Dynamixel protocol. However, before we do so, we must look at the lower-level electrical system.
+
+The goal of this guide is to give an overview of the Dynamixel communication. It is not meant to be an exhaustive list of what is already on the Robotis [manual](https://emanual.robotis.com/docs/en/dxl/protocol2/) but to give further insight into things left out of said manual, mostly the low-level electrical side, as well as any idiosyncracities. Said manual should be read after the reader has read this guide.
+
+The target-audience is somewhat broad and can be anywhere between
+
+- a team-member who is experienced with the Dynamixel system but wants to cross-reference something,
+- a new electrical, computer, or mechatronics engineering student who has had some experience with some of these concepts such as UART in ELEC2720 but may want to refresh some of the basics and needs to learn some new things such as RS485 and the Dynamixel protocol,
+- and a computer-science student who has some experience with telecommunications such as Ethernet but may have no experience at all with UART and RS485.
+
+<Alert type="warn">
+
+Assumed knowledge is a basic understanding of electrical concepts like voltage and maybe some HSC physics such as magnetic fields as well as a basic intuition for electronic devices in daily life.
```

Comment:
```suggestion
Assumed knowledge includes a basic understanding of electrical concepts (i.e. voltage), HSC physics (i.e. magnetic fields), and basic intuition for electronic devices in daily life.
```
be definitive and firm in your writing - avoid any 'maybe's. Basic intuition is a bit hard to quantify too. If this is something you think everyone probably has, then maybe just remove that part.